### DISCUSSION TITLE
background-image disapeard with my svg file (for my readme.md)

### DISCUSSION THREAD

**Discussion by Midicix** [community/community#54297] [Programming Help]
### Body

Hi !
I don't know why and how, but i modify a svg file to add a background-image
In my local computer and the friend's one, it run perfectly
But in github, the background-image doesn't load
Someone can help me ?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by multikitty** [comment]
![image](https://user-images.githubusercontent.com/124760226/235873717-d7226404-8951-46f8-97b0-901fa1c457bc.png)
which is your background?

**Reply by Midicix** [reply]
Background is cmdV2.png, but it include inside cmd.svg or cmdtest2.svg as background-image

**Comment by multikitty** [comment]
your image shows me like the following:
![image](https://user-images.githubusercontent.com/124760226/235885819-95817404-55b6-4d4e-9bf6-54bada1f1430.png)
how did you include other svg files in cmdV2.png?

And I can't find cmdV2.png in your readme file.
![image](https://user-images.githubusercontent.com/124760226/235886123-a92eabf0-e965-4142-9004-58eeacebe74a.png)

**Comment by Midicix** [comment]
i add my svg file here : https://github.com/Midicix/Midicix/blob/main/README.md?plain=1
line 3 like this picture : 
![capt1](https://user-images.githubusercontent.com/94143634/235950528-4ed2a2b4-0b18-41b9-9b8b-9c627c72fe6b.PNG)


And i have the background image in this svg file : https://github.com/Midicix/Midicix/blob/main/cmdtest2.svg?short_path=53ca6d8
Line 3 too : 
![capt2](https://user-images.githubusercontent.com/94143634/235950878-5714ebb9-18b6-4b4b-add3-7a53675ef8a2.PNG)

**Comment by Midicix** [comment]
Someone have any idea to solve this problem ?
